charts/harmony-chart/values.yaml

```diff
@@ -235,3 +235,57 @@ karpenter:
   # Node template reference. More details in https://karpenter.sh/docs/concepts/node-templates/
   nodeTemplate:
     name: "default"
+
+# Before enablin Velero, make sure to execute "velero install --crds-only" to install
+# the CRDs in the cluster.
+velero:
+  enabled: false
+
+  # For the list of supported providers, and their configuration,
+  # please visit https://velero.io/docs/main/supported-providers/
+  configuration:
+    backupStorageLocation:
+      - name: "velero-backup-harmony"
+        provider: aws
+        bucket: ""  # unique name of the bucket
+        default: true
+        config:
+          s3Url: "" # the endpoint can be any S3 compatible storage
+          region: "" # the region of the bucket
+
+    volumeSnapshotLocation:
+      - name: "velero-volume-snapshot-harmony"
+        provider: "" # provider configuration
+        config:
+          region: "" # the region of the bucket
+
+  credentials:
+    secretContents:
+      cloud: |
+        [default]
+        aws_access_key_id=""     # AWS access key ID
+        aws_secret_access_key="" # AWS secret access key
```

Comment:
I suppose keep AWS credentials in repository it's a bad idea. I'd recommend to use IAM role for service account (IRSA). For this purpose, it would be better to move the deployment of helm chart velero to terraform as part of the infrastructure. Or consider adding only the IAM role for service account (IRSA) in the terraform.

Reply:
Thank you for the clarification. 
In general, you've got the point, but I mean that the best solution is to use IRSA. To achieve this goal, it will be easier to use the terraform module https://github.com/aws-ia/terraform-aws-eks-blueprints-addons. But in general, this solution is also OK.

Reply:
I absolutely get it, but it must be noted that this is _not_ only for AWS though the naming is confusing. It should work with any s3-compatible storage. Hence, IRSA cannot be applied on those services (like DigitalOcean, Azure, or similar).